ui/src/services/router/TuringRouter.js

```diff
@@ -49,7 +49,8 @@ export class TuringRouter {
         },
         kafka_config: {
           brokers: "",
-          topic: ""
+          topic: "",
+          serialization_format: ""
```

Comment:
Do we want to be opinionated about the default serialization format? Shall we set it as `Protobuf` and save user an extra click (actually, two)?